frontend/src/general/userForm/FormConsole.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Button, Container, TextField } from "@material-ui/core";
+import PropTypes from "prop-types";
+
+export default class FormConsole extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: "",
+      password: ""
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+    this.setUsername = this.setUsername.bind(this);
+    this.setPassword = this.setPassword.bind(this);
+  }
+
+  onSubmit(event) {
+    const { handleSubmit } = this.props;
+    const { username, password } = this.state;
+    event.preventDefault();
+    handleSubmit(username, password);
+  }
+
+  setUsername(username) {
+    this.setState({
+      username
+    });
+  }
+
+  setPassword(password) {
+    this.setState({
+      password
+    });
+  }
+
+  render() {
+    const { buttonName, isValidate, isCreateUser } = this.props;
+    const { password } = this.state;
+    return (
+      <Container component="main" maxWidth="xs">
+        <div>
+          <form noValidate={!isValidate} onSubmit={this.onSubmit}>
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              id="username"
+              label="Username"
+              name="username"
+              autoComplete="username"
+              autoFocus
+              onChange={(event) => this.setUsername(event.target.value)}
+            />
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              name="password"
+              label="Password"
+              type="password"
+              id="password"
+              error={
+                isCreateUser
+                  ? password.length < 5 && password.length > 0
+                  : undefined
```

Comment:
This isn't great. Consider passing the error as a prop to FormConsole and then create a "CreateUserFormConsole" that passes the info it needs